plugins/action/jcliff.py

```diff
@@ -61,7 +61,8 @@ def _build_and_deploy_jcliff_rule_files(self, tmp_remote_src):
             'system_props': 'system-properties.j2',
             'deployments': 'deployments.j2',
             'logging': 'logging.j2',
-            'keycloak': 'keycloak.j2'
+            'keycloak': 'keycloak.j2',
+            'jms': 'jms.j2'
```

Comment:
@sabre1041 the subsystem name should be messaging (rather than JMS) and should be specific to hornetq : messaging_hornetq.

This will both show that only hornetq is supported but avoid some extras complexity when we move to support Artermis.